src/main/xerus/monstercat/api/response/Release.kt

```diff
@@ -67,4 +68,9 @@ data class Release(
 		override fun toString() = displayName
 	}
 	
-}
\ No newline at end of file
+}
+
+data class ReleaseResponse(
```

Comment:
This shouldn't be in here

Reply:
I'm not sure if creating a new file for that small data class is justified. Should we put it with ListResponse, ReleaseListResponse ?

Reply:
I moved it to a separate file and documented it
7d854bca66a6a13809c04cce1ee49f6341211561 for move
fc33ec054281b41f18226a22ede245fef0353c0d for docs

Reply:
I might have made a typo in a commit, oops